examples/pos/probabilistic.effekt

```diff
@@ -80,6 +72,17 @@ def choose3[R] { x: => R } { y: => R } { z: => R } : R / fork =
 
 type Weighted[R] { MkWeighted(weight: Double, value: R) }
 
+def showRounded(d: Double): String = show(d.round(2))
```

Comment:
I don't understand how this rounding doesn't fix the CI issue. The float inaccuracy is not reproducible on my computer.

Reply:
> Also: since when is time.effekt flaky?

No idea.. I just triggered a re-run of CI.

Reply:
Also, how does this not fail the VMTests?

Reply:
Ah you're right, I misinterpreted the obtained/expected in CI.. No idea about VMTests